### PR TITLE
bridging improvements

### PR DESCRIPTION
**Pull request by Cadair**

### CODE REVIEW

File: IDEAS.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ Google Summer Of Code Matrix Ideas, 2018 Edition!!
 - [Potential GSoC ideas:](#potential-gsoc-ideas)
   - [Contribute to Dendrite](#contribute-to-dendrite)
   - [Building Bridges!](#building-bridges)
+    - [Bridge Project Suggestions](#bridge-project-suggestions)
```
Comment (t3chguy) [inline IDEAS.md]: Bad indentation

Comment (Cadair) [reply IDEAS.md]: don't look at me the TOC is auto generated! 😆

Comment (Cadair) [reply IDEAS.md]: it is a subheading under "Building Bridges"

Comment (t3chguy) [reply IDEAS.md]: oh RIP
Ignore me

**Comment by ara4n** [comment]
thanks!

File: IDEAS.md [hunk 2]
```diff
@@ -135,20 +139,37 @@ Bridging (linking existing networks into the wider Matrix ecosystem) is one of t
...
+
+#### Bridge Project Suggestions
+
+There are a *lot* of potential projects in all the various matrix bridges, below
+is a few ideas to get you started. A good room to join if you are interested in
+working on bridges is
+[#bridges:matrix.org](https://matrix.to/#/#bridges:matrix.org)
+
+
+1. Add user puppeting support to the Slack and Gitter bridges, this would allow
```
Comment (leonerd) [inline IDEAS.md]: FYI, the `matrix-appservice-gitter` already has this feature. All that's missing is frontend UI in Riot or similar, to allow people to control it. There's even backend API hooks to tell the bridge what to do with it.

**Pull request merged**